acquia_lift_profiles/js/acquia_lift_profiles.debug.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Defines JavaScript functions needed by the Acquia Lift debugger in
+ * order to provide customer profile information.
+ */
+
+(function ($, Drupal) {
+
+  Drupal.acquiaLiftProfilesDebug = (function() {
+    var debugPrefix = "acquiaLift::debug::"
+    var curSegmentCapture = {};
+    var curSegments = [];
+    var curSegmentsOverride;
+    var curIdentities = [];
+    var message;
+    var code;
+    $(document).on("segmentsUpdated", function( event, data, capture ) {
```

Comment:
Need some commenting/documentation for this callback on what is expected in the event, data, and capture parameters.  If if is already somewhere in the Lift Web API documentation, then a link to that is fine.